client_integration_test.go

```diff
@@ -29,6 +32,9 @@ const (
 	debuglevel = "ERROR" // set to "DEBUG" for debugging
 )
 
+var spaceRegex = regexp.MustCompile(`\s+`)
```

Comment:
please move this closer to where it is used, it only looks like it is used in one test